src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -60,25 +51,9 @@ public static class Diagnostics {
 			description: "The method being called has declared that this parameter cannot receive null, but a null value is being passed."
 		);
 
-		public static readonly DiagnosticDescriptor SingletonRegistrationTypeUnknown = new DiagnosticDescriptor(
-			id: "D2L0011",
-			title: "Unable to resolve the concrete or plugin type for this registration",
-			messageFormat: "Unable to determine the concrete or plugin type for this registration; please make sure to reference the type's assembly",
-			category: "Safety",
-			defaultSeverity: DiagnosticSeverity.Error,
-			isEnabledByDefault: true,
-			description: "Singleton registrations must be known at compile-time; please make sure to reference the type's assembly."
-		);
-
-		public static readonly DiagnosticDescriptor RegistrationKindUnknown = new DiagnosticDescriptor(
-			id: "D2L0012",
-			title: "Unable to determine the kind of dependency registration attempted",
-			messageFormat: "The attempted DI registration is not known to our analysis or there was an error analyzing it. This is mostly likely because the ObjectScope is being passed as a variable, or this is a new kind of registration and needs to be handled.",
-			category: "Safety",
-			defaultSeverity: DiagnosticSeverity.Error,
-			isEnabledByDefault: true,
-			description: "All DI registrations must be known to static analyzers, to allow for thorough analysis."
-		);
+		// Retired:
+		// D2L0011 (SingletonRegistrationTypeUnknown) "Unable to resolve the concrete or plugin type for this registration"

```

Comment:
Actually dropped in 46d0599443752a888c0e1b9ac379bc3111c19634, but was straggling.